src/services/awsAuthService.js

```diff
@@ -0,0 +1,34 @@
+import {
+  AuthenticationDetails,
+  CognitoUserPool,
+  CognitoUser,
+} from 'amazon-cognito-identity-js';
+
+var authenticationData = {
+  Username: process.env.API_USER || '',
+  Password: process.env.API_PASS || '',
+};
+
+var authenticationDetails = new AuthenticationDetails(authenticationData);
+
+var poolData = {
+  UserPoolId: 'us-east-2_yWRkOEFLL', // Your user pool id here
```

Comment:
Do you think we should stick these under env variables?

Reply:
added the env